script/extract-book-v1-urls.rb

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/ruby
+
+require "open-uri"
+require 'uri'
+require 'yaml'
+
+$v1_to_v2 = [
```

Comment:
This must correspond to this table:

https://github.com/git/git-scm.com/issues/1782#issuecomment-2457407947

But how does this work? Are the values here the V2 target?  Is the V1 chapter mapped via a 2D index?  Like for V1 Ch. 1.4 Installing Git: V2 Ch. 1.5 is in the fourth position here so that seems to line up.

Reply:
This map has the v2 sections as keys, and maps to values that are arrays of v1 sections to which a particular v2 section needs to be aliased.

Reply:
> This map has the v2 sections as keys, and maps to values that are arrays of v1 sections to which a particular v2 section needs to be aliased.

Oh, scratch that, I was talking about `data/book_v1.yml`. Will add a code comment to clarify the role of `v1_to_v2`.

Reply:
Uh oh. The `$v1_to_v2` data is not even used. Which means that https://dscho.github.io/git-scm.com/book/en/Git-Tools-Submodules redirects to https://dscho.github.io/git-scm.com/book/en/v2/GitHub-Summary (i.e. 6.6 -> 6.6, when it should redirect to https://dscho.github.io/git-scm.com/book/en/v2/Git-Tools-Submodules instead, i.e. to 7.11)!

Will fix immediately.